framework/tlvf/yaml/tlvf/WSC/sM1.yaml

```diff
@@ -0,0 +1,30 @@
+#
+---
+_include: tlvf/WSC/WSC_Attributes.h
+_namespace: WSC
+
+sM1:
```

Comment:
Is it useful to define a separate struct for this? It could just as well be added directly in tlvWscM1, no?

Reply:
Well that's a tricky one.
The M1 frame belongs to WSC specs, while tlvWscM1 belongs to ieee1905. 
Do you think we should still insert the attributes directly inside the tlv? I wasn't completely sure about it.

Reply:
Good point, indeed it's better to keep it a separate struct.